configs/testing/exp_eufy_t8441

```diff
@@ -0,0 +1,6 @@
+# MODULE: t31x_sc3335_syn43436_defconfig
+FLASH_SIZE_32=y
+U_BOOT_ENV_TXT="$(BR2_EXTERNAL)/environment/eufy-t8441.uenv.txt"
+BR2_AUDIO=y
+BR2_PACKAGE_SOX=y
```

Comment:
this could probably be left out once we make it out of experimental builds, you can add it to your local build in locak.fragment